padd.sh

```diff
@@ -1100,6 +1075,14 @@ moveXOffset(){
         printf '\e[%sC' "${xOffset}"
     fi
 }
+
+# Remove undesired strings from sys_model variable - used in GetSystemInformation() function
+filterModel() {
+    FILTERLIST="To be filled by O.E.M.|To Be Filled\*|OEM\*|Not Applicable|System Product Name|System Version|Undefined|Default string|Not Specified|Type1ProductConfigId|INVALID|All Series|�"
```

Comment:
I get warnings with this regex.

```
awk: warning: escape sequence `\*' treated as plain `*'
```

Did you mean to use
```suggestion
    FILTERLIST="To be filled by O.E.M.|To Be Filled(.)*|OEM(.)*|Not Applicable|System Product Name|System Version|Undefined|Default string|Not Specified|Type1ProductConfigId|INVALID|All Series|�"
```

Reply:
Actually I just copied the strings from the [previous code](https://github.com/pi-hole/PADD/pull/306/files#diff-42c44861235d0baff7c03aea0fe6c7b7cb95f444c3a398c6d96da1290fc6bc01L262-L265).

I added the backslash because `awk` was not accepting the `*`, but now I think we don't need the asterisk, unless the string was literaly "_OEM*_".

Reply:
I don't think we'll see a litteraly `OEM*` in the wild. I  wasn't sure if you wanted to remove everything if the line started with `OEM` or just the `OEM` and leave all following characters.

Reply:
To tell the truth, I saw "To be filled by O.E.M." before, but never saw `To be filled by` and `OEM`, followed by something else.

Reply:
```
me@ryzen:~$ cat /sys/devices/virtual/dmi/id/product_family 
To Be Filled By O.E.M.
me@ryzen:~$ cat /sys/devices/virtual/dmi/id/product_name
To Be Filled By O.E.M.
```
It's a standard string in the BIOS specifications that is, well, to be replaced by the O.E.M. If don't do it, the standard string stays as it is. I doubt any manufacturer would put in something else than their real info in here. 

The result above is from a SMBIOS 3.2.0 by American Megatrends Inc. (version P1.20, release date 09/16/2020) on a ASRock FP6D4-P1 mainboard and basically nothing is set at all:
```
System Information
        Manufacturer: To Be Filled By O.E.M.
        Product Name: To Be Filled By O.E.M.
        Version: To Be Filled By O.E.M.
        Serial Number: To Be Filled By O.E.M.
        Wake-up Type: Power Switch
        SKU Number: To Be Filled By O.E.M.
        Family: To Be Filled By O.E.M.
```

This is the device (it is my local docker + Pi-hole server): https://www.asrock.com/nettop/AMD/Mars%204000U%20Series/index.de.asp?cat=

Reply:
Also, I just realize: 

`OEM*` will mach any sequence starting with "OEM" followed by zero or more "M" letters (Asterisk indicates repetition).
To remove everything after "OEM", the regex should be `OEM.*`.

Reply:
Suggestion:
1) remove everything after "To be filled by", using `To be filled.*`;
2) remove everything after "OEM", using `OEM.*`;
3) add `IGNORECASE = 1`, to remove case variations ("To be filled", "To Be Filled", "OEM", "oem", etc.).

What do you think?

Reply:
1 & 2: Why? I don't know any other variant than *exactly* `To Be Filled By O.E.M.` we want to replace. I have not even seen `OEM` myself anywhere (this doesn't, of course, mean it cannot exist)

3: Yes

Reply:
I suggested 1 & 2 because this strings were there before. As I said, I never saw this other values before.

I will remove them. We can add new values in the future, if someone finds a different string.

Reply:
Done.

I removed the strings `To be filled*` and `OEM*` and added `IGNORECASE = 1`.